platform/ios/src/MGLMapView.mm

```diff
@@ -1619,6 +1748,73 @@ - (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)twoFingerDrag
         [self notifyGestureDidEndWithDrift:NO];
         [self unrotateIfNeededForGesture];
     }
+
+}
+
+- (MGLMapCamera *)currentCameraWithEstimatedEndPoint:(CGPoint)endPoint panGesture:(UIPanGestureRecognizer *)pan
```

Comment:
If moving these methods to MGLMapCamera would be too much hassle because of dependencies on internal MGLMapView methods, we can leave the methods here, but let’s give them better names:

* `-currentCameraWithEstimatedEndPoint:panGesture:` → `-cameraByPanningWithTranslation:`
* `-currentCameraWithEstimatedZoom:` → `-cameraByZoomingToZoomLevel:`
* `-currentCameraWithEstimatedDegrees:anchorPoint:` → `-cameraByRotatingToDirection:aroundAnchorPoint:`
* `-currentCameraWithEstimatedPitch:anchorPoint:` → `-cameraByTiltingToPitch:`